winch/codegen/src/visitor.rs

```diff
@@ -36,7 +36,8 @@ macro_rules! def_unsupported {
 
                 fn $visit(&mut self $($(,$arg: $argty)*)?) -> Self::Output {
                     $($(let _ = $arg;)*)?
-                        todo!(stringify!($op))
+
+                    self.found_unsupported_instruction = Some(stringify!($op));
```

Comment:
Seems like, longer term, it would be nice if the visitor's `Output` was a `Result`. Not something we need to address now though, unless it is a pretty easy change.

Reply:
yeah I'm a bit wary to refactor most of the visitor to return `Ok(())` at the end so I'll stick to this for now

Reply:
I got to this comment while looking at Alex's comment in https://github.com/bytecodealliance/wasmtime/pull/9158/files#r1723917083 just wanted to say that this is on my radar, I had opened [this issue](https://github.com/bytecodealliance/wasmtime/issues/8096) a while ago, in order to refactor the internals to return `Result` instead of panicking, where applicable